string FileName
int8 Lap
RoiList[] Rois